app/page-objects/cloud-home-page.js

```diff
@@ -0,0 +1,119 @@
+const Page = require('./page');
+
+class CloudHomePage extends Page {
+
+    get searchButton() {
+        return $('//*[@name="q"]')
+      }
+
+      get searchResult() {
+        return $('//a[contains(.,"Google Cloud Pricing Calculator")]')
+      }
+
+      get homePageFirstFrame() {
+        return $("//iframe[contains(@src,'https://cloud.google.com/products/calculator/index_d6a98ba38837346d20babc06ff2153b68c2990fa24322fe52c5f83ec3a78c6a0.frame')]")
+      }
+
+      get homePageSecondFrame() {
+        return $('#myFrame')
+      }
+
+      get numberOfInstancesArea() {
+          return $("//input[@ng-model='listingCtrl.computeServer.quantity']")
+      }
+
+      get seriesDropdown() {
+        return $("//md-select[@placeholder='Series']")
+      }
+
+      get seriesOption() {
+        return $("//*[contains(text(), 'N1')]")
+      }
+    
+      get instanceTypeDropdown() {
+        return $("//md-select[@placeholder='Instance type']")
+      }
+
+      get instanceTypeOption() {
+        return $("//*[contains(text(), 'n1-standard-8')]")
+      }
+
+      get addGPUCheckbox() {
+        return $("//md-checkbox[@aria-label='Add GPUs']")
+      }
+
+      get GPUTypeDropdown() {
```

Comment:
Naming: only classes should start with the capitals.

Example for this element name: `gpuTypeDropdown`.

Also please check whole file and fix if it needed.